ui/test/e2e/specs/test.spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable ui-testing/no-disabled-tests */
+
+describe('DAPP Offer Up E2E Test Cases', () => {
+  context('Test commands', () => {
+    it(`should complete Keplr setup by  importing an existing wallet using 24 word phrase`, () => {
+      cy.setupWallet().then((setupFinished) => {
```

Comment:
the import-24-words stuff is happening twice when I run the test. Is that by design?